homeassistant/exceptions.py

```diff
@@ -12,6 +12,20 @@
 class HomeAssistantError(Exception):
     """General Home Assistant exception occurred."""
 
+    def __init__(self, *args: object, omit_stack_trace: bool = False) -> None:
```

Comment:
We usually type *args as Any, why is it typed as object here?

Reply:
Used auto complete

Reply:
`object` for `args` is inherited from `BaseException` .